### PR TITLE
Gles20

### PR DESCRIPTION
**Pull request by donaldalanmorrison**
The changes remove LLVM Static Analyzer Warnings/Errors (only two remain.)

### CODE REVIEW

**Comment by minggo** [comment]
Thank you.

File: cocos2dx/kazmath/src/ray2.c [hunk 1]
```diff
@@ -119,8 +119,7 @@ void calculate_line_normal(kmVec2 p1, kmVec2 p2, kmVec2* normal_out) {
 
 kmBool kmRay2IntersectTriangle(const kmRay2* ray, const kmVec2* p1, const kmVec2* p2, const kmVec2* p3, kmVec2* intersection, kmVec2* normal_out) {
     kmVec2 intersect;
-    kmVec2 final_intersect;
-    kmVec2 normal;
+    kmVec2 final_intersect = {.x = 0, .y = 0}, normal = {.x = 0, .y = 0}; // Silencing LLVM SA
```
Comment (minggo) [inline cocos2dx/kazmath/src/ray2.c:122]: this code cause compiling error on win32.

Comment (donaldalanmorrison) [inline cocos2dx/kazmath/src/ray2.c:122]: Hi Ming, sorry, I'm cloning on windows now.  It's a C99 initialization.  Whoops.

Comment (donaldalanmorrison) [inline cocos2dx/kazmath/src/ray2.c:122]: Fixed in merge request #1231. ^_^

**Pull request merged**